src/bin/mission_manager/machine.cpp

```diff
@@ -269,8 +269,10 @@ jaiabot::statechart::inmission::underway::Task::~Task()
 
     task_packet_.set_end_time_with_units(goby::time::SystemClock::now<goby::time::MicroTime>());
 
-    if (task_packet_.type() != protobuf::MissionTask::NONE)
+    if (task_packet_.type() != protobuf::MissionTask::NONE && !this->machine().rf_disable())
+    {
         intervehicle().publish<groups::task_packet>(task_packet_);
+    }
```

Comment:
You probably want to publish the task packet to interprocess so we keep logging the task packets